include/cxxopts.hpp

```diff
@@ -884,7 +884,7 @@ namespace cxxopts
       ("--([[:alnum:]][-_[:alnum:]]+)(=(.*))?|-([[:alnum:]]+)");
 
     std::basic_regex<char> option_specifier
-      ("(([[:alnum:]]),)?([[:alnum:]][-_[:alnum:]]*)?");
+      ("(([[:alnum:]]),)?[ ]*([[:alnum:]][-_[:alnum:]]*)?");
```

Comment:
True but I feel it is more clear with what is intended.  Looking at "[ ]" I'm not sure if there is something missing or not.  Would it break anything if someone used tabs or newlines?  Their code would look strange but that's probably the least of their problems.